src/core/test-unit/Globalization/UtcHelperTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using RapidCore.Globalization;
+using Xunit;
+
+namespace RapidCore.UnitTests.Globalization
+{
+    public class UtcHelperTests
+    {
+        private readonly UtcHelper utcHelper;
+
+        public UtcHelperTests()
+        {
+            utcHelper = new UtcHelper();
+        }
+
+        #region Helpers
+        private TimeZoneInfo GetNonUtcTimeZone()
+        {
+            foreach (var tzi in TimeZoneInfo.GetSystemTimeZones())
+            {
+                if (tzi.BaseUtcOffset != TimeSpan.Zero)
+                {
+                    return tzi;
+                }
+            }
+            
+            throw new InvalidOperationException("Could not find a non-utc timezone on this system");
+        }
+        #endregion
+
+        #region ToUtc - datetime input
+        [Fact]
+        public void ToUtc_datetime_alreadyUtc()
+        {
+            var given = DateTime.UtcNow;
+            var actual = utcHelper.ToUtc(given);
+            
+            Assert.Equal(given, actual);
+        }
+        
+        [Fact]
+        public void ToUtc_datetime_nonUtc_is_converted()
```

Comment:
Would be nice with a test that covers the case where the `DateTimeKind` is `unspecified` - which can happen after an ORM has persisted and loaded a date time entity

Reply:
I will add it, but it looks a little silly (which is why I left it out), as it seems that it is merely treated as "local", but also because the UtcHelper itself simply calls `.ToUniversalTime()` on the given DateTime.